app/views/admin/enterprises/_new_form.html.haml

```diff
@@ -97,10 +97,28 @@
         %input.ofn-select2.fullwidth#enterprise_address_attributes_state_id{ name: 'enterprise[address_attributes][state_id]', type: 'number', data: 'countriesById[Enterprise.address.country_id].states', placeholder: t('admin.choose'), ng: { model: 'Enterprise.address.state_id' } }
       .five.columns.omega
         %input.ofn-select2.fullwidth#enterprise_address_attributes_country_id{ name: 'enterprise[address_attributes][country_id]', type: 'number', data: 'countries', placeholder: t('admin.choose'), ng: { model: 'Enterprise.address.country_id' } }
-
+  .row
+    .three.columns.alpha
+      = af.label :latitude, t(:latitude)
+      \/
+      = af.label :longitude, t(:longitude)
+      %span.required *
+      %div{'ofn-with-tip' => t('latitude_longitude_tip')}
+        %a= t('admin.whats_this')
+    .four.columns
+      = af.text_field :latitude, { placeholder: t(:latitude_placeholder) }
+    .four.columns.omega
+      = af.text_field :longitude, { placeholder: t(:longitude_placeholder) }
+  .row
+    .three.columns.alpha
+      = "&nbsp;".html_safe
+    .five.columns.omega
+      = check_box_tag "enterprise[address_attributes][use_geocoder]"
+      = label_tag "enterprise[address_attributes][use_geocoder]", t('use_geocoder')
```

Comment:
Since `use_geocoder` is not an attribute of the enterprise or its address, it would be much easier to make this a simple parameter.

```suggestion
      = check_box_tag "use_geocoder"
      = label_tag "use_geocoder", t('use_geocoder')
```

Then we don't need to delete it from params in the controller either.